packages/backend/src/controllers/users/dto/create-user.dto.ts

```diff
@@ -1,5 +1,8 @@
 import { ApiProperty } from '@nestjs/swagger';
 export class CreateUserDto {
+  @ApiProperty()
+  name: string;
```

Comment:
lets make this optional and pull the name from the user token if it is possible. If name isn't available and hasn't been provided, throw 400 error

Reply:
> Instead of explicitly typing the `@ApiProperty({ required: false })` you can use the `@ApiPropertyOptional()` short-hand decorator.
https://docs.nestjs.com/openapi/types-and-parameters
we should probably do this across the repo but doesn't really matter